doc/users/next_whats_new/ecdf.rst

```diff
@@ -0,0 +1,4 @@
+``Axes.ecdf``
+~~~~~~~~~~~~~
+A new Axes method, `~.Axes.ecdf`, allows plotting empirical cumulative
+distribution functions without any binning.
```

Comment:
I'd suggest adding an example plot here to make it a bit more visible. (It is not every day that we add a completely new plot type...)

Subject to getting general approval of the idea etc.

Reply:
sure